### DISCUSSION TITLE
[DropdownMenu] How can I get the `ItemIndicator` status?

### DISCUSSION THREAD

**Discussion by rawnly** [radix-ui/primitives#1657] [Help]
Hello I'd like to display 2 different icons on my `CheckBoxItem` (empty box = unchecked, box with tick = checked) 
I read the docs and I can't see any chance to archive this. Am I missing something?

**Comment by benoitgrelard** [comment]
Hi @Rawnly, `CheckboxItem` is always controlled so necessarily you always have access to that state, it will be what you have passed to the `checked` prop.

**Reply by rawnly** [reply]
So i don't have to use ItemIndicator right? Or at least only for the checked state

**Reply by benoitgrelard** [reply]
It's mostly a convenience so yes if you want to fully take over the rendering here, you can!
